Java/CaesarCipherEncryption_Decryption/Decryption

```diff
@@ -0,0 +1,44 @@
+import java.util.Scanner;
```

Comment:
Just asking. Why have you added half of the files in Java folder and others in Android?

Reply:
yaa I got the issue I will resolve it and will send you.  Hope you don`t mind